src/pages/mine/index.tsx

```diff
@@ -30,7 +30,7 @@ const Content = () => {
 
     if (isLoading) return <p>Loading...</p>;
     if (error) return <p>Something went wrong! Please contact admin</p>;
-    if (tasks && tasks.tasks?.length)
+    if (tasks && tasks.length)
```

Comment:
I believe here just `tasks?.length` should work

Reply:
Yes its work